lighthouse-core/test/fixtures/fraggle-rock/reports/sample-flow-result.json

```diff
@@ -7850,7 +7850,7 @@
             "description": "Consider reducing the time spent parsing, compiling, and executing JS. You may find delivering smaller JS payloads helps with this. [Learn more](https://web.dev/bootup-time/).",
             "score": 1,
             "scoreDisplayMode": "numeric",
-            "numericValue": 274.5320000000002,
+            "numericValue": 275.90400000000034,
```

Comment:
@adamraine this looks right and the timespan wasn't looking cross frame for bootup time, but haven't verified yet. Hard to find where each step begins and ends in the big json files :)

Reply:
> Hard to find where each step begins and ends in the big json files :)

I can open an issue for an `assetSaver.saveFlowArtifacts` that separates this stuff into separate files. Wouldn't hurt to have another 800,000+ line PR on my resume 😎